pkg/internal/export/otel/traces.go

```diff
@@ -274,14 +275,30 @@ func spanKind(span *transform.HTTPRequestSpan) trace2.SpanKind {
 func (r *TracesReporter) makeSpan(parentCtx context.Context, tracer trace2.Tracer, span *transform.HTTPRequestSpan) SessionSpan {
 	t := span.Timings()
 
-	if span.TraceID != "" {
+	if len(span.TraceID) > 0 {
```

Comment:
Not really against of this form, but the idiomatic ways to check for empty strings in Go are the `== ""` and `!= ""` forms:
https://programming-idioms.org/idiom/110/check-if-string-is-blank/1155/go

Reply:
Thanks, I thought I had it reversed, but the first way was right.  I'll put it back.